spec/tagManagerWrap.spec.js

```diff
@@ -0,0 +1,185 @@
+import TagManager from '../src/tagmanager-wrap';
+
+describe('Tagmanager', () => {
+  const tagManager = new TagManager([], {
+    gtmId: 'GTM-T2HGD',
+    startPush: {
+      page_type: 'pages:demo',
+      path_category: 'reformas-e-reparos/demo',
+      experiments: window.experiments || [],
+    },
+  });
+
+  describe('constructor', () => {
+    it('defines default properties', () => {
+      expect(tagManager.options).toBeDefined();
+      expect(tagManager.dataLayer).toBeDefined();
+    });
+  });
+
+  describe('.defineProperties', () => {
+    it('sets default properties', () => {
+      expect(tagManager.options.gtmId).toEqual('GTM-T2HGD');
```

Comment:
@iondrimba Here the test is expecting to be a specific `gtmId`, we have to make it configurable.